src/main/kotlin/uk/co/craigbass/pratura/math/CurrencyExtensions.kt

```diff
@@ -4,3 +4,5 @@ import java.math.BigDecimal
 
 fun String.toDecimal(): BigDecimal = BigDecimal(this)
 fun Int.toDecimal(): BigDecimal = BigDecimal(this)
+fun BigDecimal?.toCurrencyWithSymbol() = if (this == null) "£0.00" else "£${toCurrencyString()}"
```

Comment:
Glossing over this file for now. Think there will need to be some currency/localing machinery but that will come with use cases which I don't want to worry about atm